htk-jenkins/Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM jenkins/jenkins:lts
 
-#RUN /usr/local/bin/install-plugins.sh github-api:1.111
-RUN /usr/local/bin/install-plugins.sh git ansicolor github build-name-setter timestamper kubernetes
+#RUN jenkins-plugin-cli --plugins  github-api:1.111
+RUN jenkins-plugin-cli --plugins git ansicolor github build-name-setter timestamper kubernetes
 # some issue with service; 
-RUN /usr/local/bin/install-plugins.sh maven-plugin:3.4
```

Comment:
I don't remember what forced me to use `3.4` - iirc; it was some odd issue: like some plugin was incompatible with the latest one because it was not rebuilt against it...

Reply:
Yeah something similar force me as well to remove 3.4 and update to the latest cause other plugins were not working.